### ISSUE TITLE
Changes in Default Parameters

### ISSUE THREAD

**Issue by imk1** [ENCODE-DCC/atac-seq-pipeline#136]
I noticed that you made some substantial changes in the default parameters on May 16: https://github.com/ENCODE-DCC/atac-seq-pipeline/commit/fd6c21b4bd97ac3323c00ae3cc1218e6dd5d75ef#diff-861c79f8024bbde6ed781eaea1f172c6.

1.  Am I correct in thinking that you changed the multi-mapping parameter from 0 to 4 because not allowing for any multi-mappers was causing some TSS's from known expressed genes in various datasets to not have peaks?

2.  What was your reasoning for changing the smoothing window from 150 to 73?

3.  What was your reasoning for changing the IDR threshold from 0.1 to 0.05?

In the future, it would be great if you could post an announcement when default parameters change.  A lot of people I know are running the pipeline on datasets they want to compare and pulling before every time they run it, assuming that changes are bug fixes and not realizing that changes might affect the results they are getting.  Thank you very much!

**Comment by leepc12** [comment]
We are very sorry about not announcing the change of important default parameters.

**Comment by akundaje** [comment]
Yes. Definitely a good idea to clearly announce changes in parameters.

Irene - Regarding your questions

_1. Am I correct in thinking that you changed the multi-mapping parameter from 0 to 4 because not allowing for any multi-mappers was causing some TSS's from known expressed genes in various datasets to not have peaks?_

Yes. Correct. The modest multimapping prevents holes at important regions without causing artifacts. Using only unique mapping reads does not change results significantly, but will have a few such mappability holes at some important regions.

_2. What was your reasoning for changing the smoothing window from 150 to 73?_

The goal was to improve resolution of peaks. However, we recently noticed that 73 bp smoothing infact starts causing artifacts in peak calling in some circumstances likely because the peak caller starts picking up flanking nucleosomes instead of just the accessible sites. So we are switching back to default 150. I would suggest manually switching the smoothing to 150 until that happens.

_3. What was your reasoning for changing the IDR threshold from 0.1 to 0.05?_

We found 0.1 to be a bit too relaxed for high quality datasets. We had initially set it to 0.1 since ATAC-seq data at the time was on average moderate quality. Since then with newer protocols, quality has significant improved. Since we anyway use naive overlap peaks as the relaxed version, we  decided to switch to 0.05 in order to be more stringent. 0.1 will not affect things very significantly. You will just have some more lower confidence peaks.
